earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/core/glc_assembler.py

```diff
@@ -196,11 +208,21 @@ def ExtractFileFromGlx(self, glc_path, file_in_glc, output_file):
 
   def ExtractFilelistFromGlx(self, glc_path, regex):
     """Extract given file from glc file and save it to the given path."""
+    logger = self.GetLogger()
     os_cmd = ("%s/geglxinfo "
               "--glx=\"%s\" "
               "--list_files"
               % (COMMAND_DIR, glc_path))
     glx_info = utils.RunCmd(os_cmd)
+    if not isinstance(glx_info, list) or len(glx_info) == 0:
```

Comment:
I don't think there's any point in guarding against the result of `utils.RunCmd()` not being a list.  We wrote that method to return a list, if it doesn't we have a bug in our code, and should get an exception.

Giving the user a message that says "Unexpected empty results running command" is probably just going to confuse people to think they need to try executing the command in a different way, rather than telling them there's a bug in the Python code.